### ISSUE TITLE
current version does not work wwith -O3 on some compilers

### ISSUE THREAD

**Issue by alexriegler12** [mnurzia/rv#4]
if I compile rv.c with MinGW separately with -O3 enabled and then link it to my SDL Emulator, it doesn't work.
This does not occur with previous versions.

**Comment by alexriegler12** [comment]
After disabling the endcvt function, it works fine.

**Comment by mnurzia** [comment]
Could you provide more details on why it doesn't work? Does it segfault?

**Comment by alexriegler12** [comment]
No, it does not segfault, but just doesn't execute my program correctly

**Comment by mnurzia** [comment]
You need to be more specific, otherwise I can't fix the problem. What is the program being run?
